src/ex1/app.js

```diff
@@ -0,0 +1,152 @@
+
+
+window.addEventListener('load',()=> {
+  const form = document.querySelector('#new-task-form');
+  const input = document.querySelector('#new-task-input');
+  const filterOption = document.querySelector(".filter-todo");
+  const list_el = document.querySelector('#tasks');
+  var audio = new Audio('checksound1.mp3');
+  var today = new Date();
+  var date = today.getFullYear()+'-'+(today.getMonth()+1)+'-'+today.getDate();
```

Comment:
where do you use this date? I didn't see an date related info on the application